### PR TITLE
search bar on all pages, cleaned up duplication, added filter tag on …

### PR DESCRIPTION
**Pull request by heron2014**
Search bar appears on all pages, some css clean up, filter for name and skill

### CODE REVIEW

**Comment by nelsonic** [comment]
I've re-started the Travis Build to check if there's a timeout issue causing the fail ...

**Comment by heron2014** [comment]
Do you want me to amend this or you will do it?

**Comment by iteles** [comment]
@heron2014 Makes sense for you to finalise the work on this branch :blush:

**Comment by heron2014** [comment]
ok :)

**Comment by nelsonic** [comment]
@iteles are you checking this out?

File: assets/css/enhanced.css [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
-  display: inline;
-}
   #search-btn {
       width: 100px;
   }
   #search-box {
       width: auto;
   }
+  #filter {
+    padding: 9px 15px;
```
Comment (iteles) [inline assets/css/enhanced.css]: As agreed @heron2014, we'll add the mix of `em` and `px` sizing as a bug for the next iteration given our end of sprint priorities.

**Comment by iteles** [comment]
@nelsonic Yes.

@heron2014 I like the direction this is going in! There are still a few things to do around cleaning up of unused things (such as lineicons), researching exactly how media queries load and tightening up the CSS selectors, but we will open issues on this as they are enhancements and not bugs.

**Pull request merged**